lib/features/docker_exec_websocket_server.js

```diff
@@ -0,0 +1,43 @@
+var debug = require('debug')('docker-worker:features:docker_exec_websocket_server.js');
+var DockerExecServer = require('docker-exec-websocket-server').DockerExecServer;
+// var waitForPort = require('../wait_for_port');
+var http = require('http');
+var slugid = require('slugid');
+
+export default class WebsocketServer {
+  constructor () {
+    this.path = slugid.v4();
+  }
+
+  async created (task) {
+    debug('creating ws server');
+    //TODO: add https support
+    var httpServ = http.createServer();
+    var port;
+    //searching for an open port between 32768 and 61000
+    for (let i = 0; i < 100; i++)
+    {
```

Comment:
nit, but line break is here is a bit excessive...
Anyways, I wouldnt use a for-loop they are bad because once you pass `100` attempts it just continues without throwing an error... That's not good...

I would do:
```js
var attempts = 0;
while (true) {
  port = ...
  try {
    ....
  } catch (err) {
    if (err.code !== 'EADDRINUSE') {
      throw err;
    }
    attempts += 1;
    if (attempts >= MAX_RANDOM_PORT_ATTEMPTS) {
      throw err;
    }
  }
}
```

Then at the top of the file declare:
```js
const MAX_RANDOM_PORT_ATTEMPTS = 20;
```

Btw, love that you remove event handler for `error` on success...